static/app/views/dashboardsV2/widgetCard/widgetCardChartContainer.tsx

```diff
@@ -76,6 +76,12 @@ export function WidgetCardChartContainer({
                 errorMessage={errorMessage}
                 widget={widget}
                 organization={organization}
+                isMobile={isMobile}
```

Comment:
We don't need these on the issues widget card, do we? The other ones need these because they're doing stuff for charts but Issues widgets just renders the table for now